Firefox should not be compiled with -fomit-frame-pointer to allow better core file debugging.

--- firefox-140.1.0/build/moz.configure/toolchain.configure
+++ firefox-140.1.0/build/moz.configure/toolchain.configure
@@ -3078,6 +3078,7 @@ or debug
         or msan
         or asan
         or ubsan
+        or (target.os == 'SunOS')
         or (target.os == "WINNT" and target.cpu in ("x86", "aarch64"))
         or target.os == "OSX"
     )
